src/routes/console/organization-[organization]/settings/deleteOrganizationModal.svelte

```diff
@@ -73,9 +75,16 @@
         </p>
     {:else}
         <p data-private>
-            Are you sure you want to delete <b>{$organization.name}</b>? All projects ({$projects.total})
-            and data associated with this organization will be deleted. This action is irreversible.
+            Are you sure you want to delete <b>{$organization.name}</b>?
+            {#if $projects.total > 0}
+                All projects ({$projects.total}) and data associated with this organization will be
+                deleted.
+            {:else}
+                There are <b>no projects</b> associated with this organization.
+            {/if}
         </p>
+
+        <p><b>This action is irreversible.</b></p>
```

Comment:
cc design team about this change.

Currently, modal just has a copy, and says 0 projects in case it's 0.

Suggestion here is to have separate copy for when there is 0 projects. Inteiton, I believe, is to scare the developer a bit less in such scenario.

I personally like this change, but I would love to hear from design team to see if we can make it **more** scare for when there are projects, compared to when there are 0 proejcts.

Reply:
Thanks for the elaboration. I'll wait in the meantime for design team's feedback.

Note: In order to make this a bit more scary, we could increate the text size of `This action is irreversible`. I am not sure if using a brighter color would look good though. Current color scheme seems fine enough.